src/components/sizeFilter/SizeFilter.jsx

```diff
@@ -0,0 +1,37 @@
+import { Title, List, ListItem } from './styles';
+import SIZES from 'constants/SIZES';
+
+const sizesKeys = Object.keys(SIZES);
+
+const SizeFilter = ({ activeSizes, setActiveSizes }) => {
+  const toggleSize = (size) => {
+    if (activeSizes.includes(size)) {
+      setActiveSizes(
+        activeSizes.filter((activeCategory) => activeCategory !== size)
+      );
+    } else {
+      setActiveSizes([...activeSizes, size]);
+    }
+  };
+
+  return (
+    <>
+      <Title>Size</Title>
+      <List>
+        {sizesKeys.map((sizeKey) => {
```

Comment:
```suggestion
        {Object.entries(SIZES).map(([k, v]) => {
```
или импортируй сразу готовый вариант

Reply:
Изменил на
```suggestion
        {Object.entries(SIZES).map(([k, size]) => {
```